pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.1</version>
+        <version>2.19</version>
```

Comment:
The real reason was to bump surefire plugin up to 2.19.1 so testing could be done with -Dtest=method1#method2. Will revert the change